packages/coordinator/src/components/Schedule/ScheduleWeek/ScheduleWeekComponent.module.scss

```diff
@@ -0,0 +1,46 @@
+.schedule {
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  background-color: var(--coordinator-background);
+  position: relative;
+}
+
+.dayLabel {
+  flex: 1;
+  text-align: center;
+  color: var(--coordinator-gray);
+}
+
+.dayNumber {
+  font-weight: 500;
+  font-size: 16px;
+}
+
+.dayLetter {
+  font-size: 14px;
+}
+
+.scheduleWeek {
+  overflow-y: auto;
+  width: 100%;
+  display: grid;
+}
```

Comment:
maybe hide the scrollbar? it is probably best to ask Matar..

Reply:
It hidden by default on mobile, you only see it on web